doc/source/myrocks/added-features.rst

```diff
@@ -0,0 +1,21 @@
+.. _added-features:
+
+================================================================================
+Updated Supported Features
+================================================================================
+
+The following is a list of the latest supported features:
+
+* **Percona Server for MySQL** 8.0.27-17 adds support for ``SELECT FOR UPDATE SKIP LOCKED/NOWAIT``. The transaction isolation level must be ``READ COMMITTED``.
```

Comment:
Please @patrickbirch watch out that there is being an intermediate release 8.0.26-17 so the next one is 8.0.27-18.

I guess this changes are included starting at 8.0.26-17 but please, could you verify it asking @percona-ysorokin ?